docs/index.md

```diff
@@ -10,4 +10,11 @@ I also want to note that just because I've learned that something is the most ef
 
 If you have a question, [open an issue](https://github.com/evykassirer/playing-the-internship-game/issues/new) with it and I'll try to add the answer in here somewhere. If you'd like to ask a question anonymously, or just less publicly, you can also fill out [this form](https://goo.gl/forms/jdmIcCBQcbBf5P5w1)
 
-If you have advice to add, please feel free to open a pull request!
\ No newline at end of file
+If you have advice to add, please feel free to open a pull request!
+
+### Creators
+
+- [Evy Kassirer](http://www.evykassirer.com/)
```

Comment:
can you put 
```
Content created by [Evy Kassirer](http://www.evykassirer.com/)
Also viewable at [this website](https://evykassirer.github.io/playing-the-internship-game)
``` 
at the top of the readme

and then at the bottom 

`Website generated by [Ronak Shah](https://ronakshah.net) with [mkDocs](http://www.mkdocs.org/)`

how's that sound?